client/src/utils/auth/index.js

```diff
@@ -15,3 +19,20 @@ export function getAuthToken() {
 export function isLoggedIn() {
   return Boolean(getAuthToken());
 }
+
+/**
+ * Get token from backend API and store it in local storage
+ */
+export function login(argemail, argpassword) {
+  superagent.post('http://localhost:5000/api/v0/auth/tokens')
+    .send({ email: argemail, password: argpassword })
+    .then(
+      function success(res) {
+        window.localStorage.OneLoveAuthToken = res.body.token;
+        console.log(res.body.token);
+      },
+      function error(err) {
```

Comment:
Use `.catch` for error handling instead of error callback, see http://pouchdb.com/2015/05/18/we-have-a-problem-with-promises.html (Rookie mistake #3: forgetting to add .catch() part)